src/components/auth/SignUpScreen.js

```diff
@@ -1,290 +1,303 @@
-      // grab user unique sub
-      console.log('sign up successful with result: ', data)
-      this.setState({data})
-      this.setState({userSub: data.userSub})
-      console.log(this.state.userSub)
-      Alert.alert(i18n.t('An email has been sent to confirm your sign up'))
-      this.saveUser()
-    })
-    .catch(err => {
-      this.setState({ isLoading: false })
-      if (! err.message) {
-        console.log('Error when signing up: ', err)
-        Alert.alert(i18n.t('Error when signing up: '), err)
-      } else {
-        console.log('Error when signing up: ', err.message)
-        Alert.alert(i18n.t('Error when signing up: '), err.message)
-      }
+      },
     })
+      .then((data) => {
+        // grab user unique sub
+        // console.log("sign up successful with result:", data);
+        this.setState({ data });
+        this.setState({ userSub: data.userSub });
+        // console.log(this.state.userSub);
+        Alert.alert(i18n.t("An email has been sent to confirm your sign up"));
+        this.saveUser();
+      })
+      .catch((err) => {
+        this.setState({ isLoading: false });
+        if (!err.message) {
+          // console.log("Error when signing up: ", err);
+          Alert.alert(i18n.t("Error when signing up: "), err);
+        } else {
+          // console.log("Error when signing up: ", err.message);
+          Alert.alert(i18n.t("Error when signing up: "), err.message);
+        }
+      });
   }
 
   // adds user to Amplify database
   async saveUser() {
     let newUser = {
       body: {
-        "UserId": this.state.userSub,
-        "TreesPlanted": 0,
-      }
-    }
+        UserId: this.state.userSub,
+        TreesPlanted: 0,
+      },
+    };
+
     const path = "/Users";
 
     // Use the API module to save the note to the database
     await API.put("ZeroCarbonREST", path, newUser)
-      .then(apiResponse => {
-        this.setState({apiResponse});
-        console.log("Response from saving user: " + apiResponse);
-        this.setState({ isLoading: false })
-        this.props.navigation.navigate('SignIn')
+      .then((apiResponse) => {
+        this.setState({ apiResponse });
+        // console.log("Response from saving user: " + apiResponse);
+        this.setState({ isLoading: false });
+        this.props.navigation.navigate("SignIn");
       })
-      .catch(e => {
-      console.log(e);
-    })
+      .catch((e) => {
+        console.log(e);
+      });
   }
-  
+
   render() {
     return (
-      <SafeAreaView style={styles.container}>
-        <KeyboardAvoidingView style={styles.container} behavior='padding' enabled>
-          <TouchableWithoutFeedback
-            style={styles.container}
-            onPress={Keyboard.dismiss}>
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+        <View style={styles.backDrop}>
+          <KeyboardAvoidingView
+            style={styles.keyboardView}
+            behavior={Platform.OS == "ios" ? "position" : "position"}
```

Comment:
set android behavior to null (works better for some reason)

Reply:
Android behavior cannot be null. When specifying **keyboardVerticalOffset** there must be a behavior specified (**keyboardVerticalOffset** forces android to behave similar to IOS). Also I found it as best practice to specify the behavior so I can control it. Can you send me an example of how the keyboard is functioning on your device?

Reply:
https://reactnative.dev/docs/0.57/keyboardavoidingview#behavior

And note your REF #3

Reply:
These are two different screens. The link you provided states it may help but in this case it does not. It actually breaks the keyboardAvoidingView's functionality on Android while it works flawlessly on IOS. So I had to force the behavior to occur. Otherwise with multiple inputs it gets nasty trying to to get this finicky element to work appropriately.

Reply:
Okay yea, I totally should've taken the time and actually tried this out myself first. You're right on this one. Marking this as resolved.